### ISSUE TITLE
Mono 3.2.2 Builds failing, as template doesn't exist.

### ISSUE THREAD

**Issue by m50** [aBARICHELLO/godot-ci#29]
Hi, I am receiving this message in my builds:

> mv: cannot stat '/root/.local/share/godot/templates/3.2.2.stable': No such file or directory

I am using the `barichello/godot-ci:mono-3.2.2` container. 

That folder doesn't exist.

```
root@99379c0bcc4a:~/.local/share/godot/templates# ls
3.2.2.beta3.mono
```

While that's not an issue, because I can change it, it does mean that there is an issue in your template. If nothing else, maybe mention it in the instructions, so that people know that they may need to edit this?

**Comment by asheraryam** [comment]
Hey, thanks for the tip, seems the `mono-3.2.2` tag was built from an [older commit](https://github.com/aBARICHELLO/godot-ci/blob/22f6aa9217d8edbec82fb4d9f06ce6857b8d6086/mono.Dockerfile) that uses 3.3.2 befa3.

We will be restarting the docker build for this tag so will be corrected very soon.

In the meantime, the `mono-latest tag` is correctly using 3.2.2 stable so you can switch to that if you'd like.

**Comment by aBARICHELLO** [comment]
Triggered a new build, should be fixed in half an hour or so.

**Comment by asheraryam** [comment]
I noticed the build is using the same commit hash. Seems the 3.2.2 branch is out of date, it did not get rebased on the mono changes from master. 

![image](https://user-images.githubusercontent.com/43449832/91098743-b2dd7080-e66a-11ea-8ecd-41f681a7bd01.png)

I added this pull request now to fix the issue. https://github.com/aBARICHELLO/godot-ci/pull/30

I am not sure if this triggers the builds on dockerhub, since I cannot see ongoing builds, only completed ones, so it might need to be triggered manually.
